app/config/FeatureSwitches.scala

```diff
@@ -25,11 +25,9 @@ import javax.inject.{Inject, Singleton}
 trait FeatureSwitches {
 
   def isIfsEnabled: Boolean
-
   def isEndpoint2089Enabled: Boolean
-
   def isEnabled(key: String): Boolean
-
+  def isScp005a_QuarterlyTypeChoiceEnabled: Boolean
```

Comment:
Can we move this above `isEnabled`? Makes sense to keep similar functions grouped together